### PR TITLE
Portfolio page

### PR DESCRIPTION
**Pull request by wale-prog**
A portfolio and About me pages have been added to be merged with the introduction page earlier added to the main branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,19 +64,124 @@ <h3>
...
+    <div class="grid-container">
 
-            <ul class="socials container">
-                <li><a href="#"><img src="images/github.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="images/linkedin2.png" alt="linkedin icon"></a></li>
-                <li><a href="#"><img src="images/two.png" alt=""></a></li>
-                <li><a href="#"><img src="images/twitter.png" alt="twitter icon"></a></li>
-                <li><a href="#"><img src="images/medium.png" alt="medium logo"></a></li>
+        <section id="portfolio">
+            <p class="portfolio-heading">My Recent Work</p>
```
Comment (TedLivist) [inline index.html:83]: - [ ] In keeping with semantic principles, this header should be appropriately wrapped using the **`hx`** tag, not **`p`** tag. Kindly refactor.

File: index.html [hunk 2]
```diff
@@ -59,19 +64,124 @@ <h3>
...
+    </div>
+
+    <section id="about-me">
+        <h1 class="container">About me</h1>
+        <p class="container">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.</p>
+        <a href="#" class="btn skill-list-btn container">Get My Resume</a>
+        <img src="images/Group.png" alt="">
```
Comment (TedLivist) [inline index.html:146]: - [ ] Decorations should be background-images (in CSS), not `img` tags. Kindly remove this image here and implement the decoration using CSS.

File: style.css [hunk 3]
```diff
@@ -101,13 +101,245 @@ body {
...
+  width: 327px;
+  padding: 12px 118.7px;
+  height: 48px;
+  text-decoration: none;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  line-height: 24px;
+  font-size: 16px;
+}
```
Comment (TedLivist) [inline style.css:224]: - [ ] Kindly ensure that your button color effects for hover, active (pressed) and disabled are all implemented as we have in the Figma design

  buttons-color

File: index.html [hunk 4]
```diff
@@ -59,19 +64,124 @@ <h3>
...
+
+    </div>
+
+    <section id="about-me">
+        <h1 class="container">About me</h1>
+        <p class="container">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.</p>
+        <a href="#" class="btn skill-list-btn container">Get My Resume</a>
```
Comment (TedLivist) [inline index.html:145]: - [ ] Kindly ensure that the button effects highlighted in the design are implemented.

  buttons-color

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @wale-prog,

Good job so far!
✔️ Good documentation
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -59,18 +64,129 @@ <h3>
...
-            <ul class="socials container">
-                <li><a href="#"><img src="images/github.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="images/linkedin2.png" alt="linkedin icon"></a></li>
-                <li><a href="#"><img src="images/two.png" alt=""></a></li>
-                <li><a href="#"><img src="images/twitter.png" alt="twitter icon"></a></li>
-                <li><a href="#"><img src="images/medium.png" alt="medium logo"></a></li>
+    <div class="grid-container">
+
+        <section id="portfolio">
+            <h1 class="portfolio-heading">My Recent Work</h1>
```
Comment (mateo951) [inline index.html:83]: Addressed ✔

File: style.css [hunk 6]
```diff
@@ -101,13 +99,274 @@ body {
...
+  padding: 12px 118.7px;
+  height: 48px;
+  text-decoration: none;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  line-height: 24px;
+  font-size: 16px;
+  background-color: #ff6b00;
+}
```
Comment (mateo951) [inline style.css:248]: Addressed 🚀

**Review by mateo951 — APPROVED** [review]
### Hi @wale-prog,

Nice job making all the necessary changes requested by the previous reviewer 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**